static/elements/chromedash-featurelist.js

```diff
@@ -0,0 +1,387 @@
+import {LitElement, html} from 'https://unpkg.com/@polymer/lit-element@latest/lit-element.js?module';
+import '/static/elements/chromedash-feature.js';
+
+class ChromedashFeaturelist extends LitElement {
+  static get properties() {
+    return {
+      whitelisted: {type: Boolean}, // From attribute
+      features: {type: Array, attribute: false}, // Directly edited and accessed in template/features.html
+      metadataEl: {type: Object}, // The metadata component element. Directly edited in template/features.html
+      searchEl: {type: Object}, // The search input element. Directly edited in template/features.html
+      filtered: {type: Array},
+    };
+  }
+
+  constructor() {
+    super();
+    this.features = [];
+    this.filtered = [];
+    this.metadataEl = document.querySelector('chromedash-metadata');
+    this.searchEl = document.querySelector('.search input');
+    this.whitelisted = false;
+    this._hasInitialized = false; // Used to check initialization code.
+    this._hasScrolledByUser = false; // Used to set the app header state.
+
+    this._featuresUnveilMetric = new Metric('features_unveil');
+    this._featuresFetchMetric = new Metric('features_loaded');
+    this._featuresUnveilMetric.start();
+    this._featuresFetchMetric.start();
+
+    this._loadData();
+  }
+
+  _loadData() {
+    const featureUrl = location.hostname == 'localhost' ?
+      'https://www.chromestatus.com/features_v2.json' : '/features_v2.json';
+
+    fetch(featureUrl).then((res) => res.json()).then((features) => {
```

Comment:
Can you use async functions and await? This would be more readable, IMO.

Reply:
Done